doc/configuring.rst

```diff
@@ -54,33 +62,29 @@ Below is an example of the configuration file in its most basic form:
 
     configs = Variations([dict(name="Build", command=["build.sh"])])
 
-This configuration file uses a :class:`Variations` class
-from the :mod:`universum.configuration_support`
-module and defines one build configuration.
+This configuration file uses a :class:`Variations` class from the :mod:`universum.configuration_support`
```

Comment:
Do we plan to use ProjectConfiguration in the documentation?

Reply:
Yes. I think clients should switch from Variations([dict(foo=bar)]) to Variations([ProjectConfiguration(foo=bar)]) for added type safety in their own projects. This substitution can already be done seamlessly.